projects/vdk-core/src/vdk/internal/builtin_plugins/config/vdk_config.py

```diff
@@ -64,6 +64,7 @@ def vdk_configure(self, config_builder: ConfigurationBuilder) -> None:
         config_builder.add(
             LOG_LEVEL_VDK,
             None,
+            True,
```

Comment:
wasn't the default value true?

```
def add(
        self,
        key: ConfigKey,
        default_value: ConfigValue,
        show_default_value=True,
        description=None,
    ) 
```

Reply:
Yes, but, but I was setting show_default_value = "description" due to order of parameters.